generative_ai/prompts/prompt_load_or_retrieve.py

```diff
@@ -29,17 +30,28 @@ def load_prompt_generate() -> str:
     # Initialize vertexai
     vertexai.init(project=PROJECT_ID, location="us-central1")
 
+    # Create local Prompt
+    prompt = Prompt(
```

Comment:
How about we call this `local_prompt` and rename `get_prompt` at L46 to `prompt`?  I think `get_prompt` as a variable name is a bit clunky.